src/index.ts

```diff
@@ -67,6 +74,27 @@ function main() {
         })
       ),
     });
+
+    context.subscribe("action.execute", ({ action, args, ...yup }) => {
+      if (action === "footpathplace" && settings.asYouGo) {
+        console.log(yup);
+        const { x, y, z } = args as FootpathAdditionPlaceArgs;
+        const addition = findAddition(
+          settings.bench,
+          settings.bin,
+          x / 32,
+          y / 32
+        );
+
+        context.executeAction(
```

Comment:
This technically works, but the action is spammed and you can't tell when the "placement" actually occurs:

![image](https://user-images.githubusercontent.com/113026/155237180-7284f0c3-1f0f-4f8f-9935-41f4b83784a7.png)

Simply holding your cursor above a tile with the path tool selected will eventually drain your entire bank account, because it "places" the addition when the footpath isn't actually placed yet. Not sure if this is a bug or not but I brought it up [in Discord](https://discord.com/channels/264137540670324737/696130778618396683/945820918217855006) so hopefully we'll get some clarification.